modin/pandas/indexing.py

```diff
@@ -39,23 +40,22 @@
 from .utils import is_scalar
 
 
+# TODO: rename "scaler" to scalar everywhere in this file
+
+
 def is_slice(x):
```

Comment:
I wonder if we really need this function...

Reply:
Don't worry. It is not used anywhere :)